src/jbake/content/officers-and-board.html

```diff
@@ -9,42 +9,87 @@
+                        <a tabindex="0" class="btn btn-lg btn-primary" role="button" data-toggle="popover" data-html=true data-trigger="focus" data-placement="bottom" data-content="">Shashank Bharadwaj</a>
+                    </div>
+                </div> 
+                
+            </div> 
+
         </div>
 
         <div class="col-lg-3">
             <div class="card text-white bg-primary mb-3" style="max-width: 15rem;">
                 <div class="card-body">
                     <h5 class="card-title" style="text-align: center">Chairman of the Board</h5>
-                    <img src="images/cjug_small.jpeg" onload="this.onload=null; this.src=gravatar('jonathan.ross@cjug.org', '150');" style="display: block;margin-left: auto;margin-right:auto"/>
+                    <img src="images/cjug_small.jpeg" onload="this.onload=null; this.src=gravatar('bob.paulin@cjug.org', '150');" style="display: block;margin-left: auto;margin-right:auto"/>
                     <br/>
                     <div style="text-align:center">
-                        <a tabindex="0" class="btn btn-lg btn-primary" role="button" data-toggle="popover" data-html=true data-trigger="focus" data-placement="bottom" data-content="<p>Jonathan Ross is a senior Java/C++ developer and financial engineer atIMC Financial Markets (a proprietary trading firm based in Amsterdam,with offices in New York, Chicago, and Sydney.)  Having programmed Java since 1997, he knows a think or two about Java and the Java VirtualMachine.</p><p>Core Java is his thing; concurrency, optimizing code and JVM tuning his passion; he has a soft-spot for good type systems and functional programming.  A JCP member, he recently started tinkering in theOpenJDK source code and speaking at major Java conferences.  He is currently seriously considering Kotlin as his next production language.</p><p>Outside work, when not learning a new programming language, Jonathan makes as much music as he can.  He plays drums in a local Alt-Rock duo,and has been known to Jam with the CJUG president as a member of the NullPointers, the one-and-only Java Community Band.</p>">Jonathan Ross</a>
+                        <a tabindex="0" class="btn btn-lg btn-primary" role="button" data-toggle="popover" data-html=true data-trigger="focus" data-placement="bottom" data-content="Bob Paulin is an independent consultant and speaker that has been developing open source software for the past 10 years. Bob has presented at large international conferences such as ApacheCon, JavaOne and Devnexus. He frequently shares his knowledge and opinions on the Java Pub House and Jave Off Heap podcasts. Bob is actively involved in the Chicago Java User Group (CJUG) and CJUG4Kids presenter/organizer. Bob is a passionate member of the ASF and donates his time to Apache Felix and Apache Tika as a committer and PMC respectively. When not coding, Bob enjoys coaching football, robotics, and spending time with his wife and 4 kids.">Bob Paulin</a>

```

Comment:
Looks like I misspelled java as Jave. I can correct this as well but if you have a moment it would be appreciated!